source/extensions/filters/common/ratelimit/ratelimit_impl.cc

```diff
@@ -1,20 +1,35 @@
-#include "common/ratelimit/ratelimit_impl.h"
+#include "extensions/filters/common/ratelimit/ratelimit_impl.h"
 
 #include <chrono>
 #include <cstdint>
 #include <string>
 #include <vector>
 
 #include "envoy/api/v2/ratelimit/ratelimit.pb.h"
+#include "envoy/registry/registry.h"
```

Comment:
This is left over from my singleton experiments. Will cleanup if we decide not to use singleton.